From 85763549c53b2907dab094163f1404b2233f8029 Mon Sep 17 00:00:00 2001
From: Adam Duskett <aduskett@gmail.com>
Date: Mon, 9 Oct 2017 16:51:20 -0400
Subject: [PATCH] Add DESTDIR to setfiles

The addition of this patch makes the use of DESTDIR
mandatory as there are conditional checks which would fail if it's not
defined.

This patch was updated from the patch provided by Niranjan Reddy to
accomodate version 2.5

Signed-off-by: Clayton Shotwell <clayton.shotwell@rockwellcollins.com>
Signed-off-by: Niranjan Reddy <niranjan.reddy@rockwellcollins.com>
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
[Update for 2.8]
Signed-off-by: Adam Duskett <aduskett@gmail.com>
[Update for 3.2]
---
 setfiles/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setfiles/Makefile b/setfiles/Makefile
index a3bbbe1..df675cf 100644
--- a/setfiles/Makefile
+++ b/setfiles/Makefile
@@ -3,7 +3,7 @@ LINGUAS ?= ru
 PREFIX ?= /usr
 SBINDIR ?= /sbin
 MANDIR = $(PREFIX)/share/man
-AUDITH ?= $(shell test -f /usr/include/libaudit.h && echo y)
+AUDITH ?= $(shell test -f $(DESTDIR)$(PREFIX)/include/libaudit.h && echo y)
 
 CFLAGS ?= -g -Werror -Wall -W
 override LDLIBS += -lselinux -lsepol
-- 
2.13.6

